README.md

```diff
@@ -1798,4 +1805,4 @@ P.S that's Riziy and he help me study ;)
 ![Riziy kitten](assets/images/riziy-vcheryasova.jpg)
 
 # Hi everyone! :+1:
-[!cat](gif/kitty2.gif)
\ No newline at end of file
+[!cat](gif/kitty2.gif)
```

Comment:
I think - you made a mistake here - 
``` [!cat](gif/kitty2.gif) ```
The sign "!" must be before brackets. That"s why your picture is not displayed.
You can check and compare with examples of other students https://github.com/kottans/mock-repo/pull/537/files